### PR TITLE
[Durable Objects] Add new WebSocket Hibernation API methods

### PR DESCRIPTION
**Pull request by cacheburner**
We introduced the following methods:
- state.getTags()
- state.getHibernatableWebSocketEventTimeout()
- state.setHibernatableWebSocketEventTimeout()

DO NOT MERGE until next release of Workers goes out.

---

Re-opening the PR because it didn't add the rendered docs.

### CODE REVIEW

File: content/durable-objects/api/websockets.md [hunk 1]
```diff
@@ -143,9 +143,19 @@ These methods are part of the [Hibernatable WebSockets API](/durable-objects/ref
...
+
+{{</definitions>}}
+
+### getTags
+
+{{<definitions>}}
+
+- {{<code>}}getTags(ws{{<param-type>}}WebSocket{{</param-type>}}){{</code>}} : {{<type>}}Array\<string>{{</type>}}
+
+  - Returns an Array of tags associated with the given WebSocket. Throws if you have not called `state.acceptWebSocket()` on the given WebSocket.
```
Comment (Maddy-Cloudflare) [inline content/durable-objects/api/websockets.md]: ```Throws if you have not called `state.acceptWebSocket()` on the given WebSocket.```

"Throws" what? Can you be more specific here?

File: content/durable-objects/api/websockets.md [hunk 2]
```diff
@@ -189,6 +199,28 @@ These methods are part of the [Hibernatable WebSockets API](/durable-objects/ref
 
 {{</definitions>}}
 
+### setHibernatableWebSocketEventTimeout
+
+{{<definitions>}}
+
+- {{<code>}}setHibernatableWebSocketEventTimeout(timeout{{<param-type>}}number{{</param-type>}}{{<prop-meta>}}optional{{</prop-meta>}}){{</code>}} : {{<type>}}void{{</type>}}
+
+  - Sets the maximum amount of milliseconds a WebSocket event (see handler methods below) can run for.
```
Comment (Maddy-Cloudflare) [inline content/durable-objects/api/websockets.md]:
```suggestion
  - Sets the maximum amount of milliseconds a WebSocket event (refer to the handler methods below) can run for.
```

File: content/durable-objects/api/websockets.md [hunk 3]
```diff
@@ -189,6 +199,28 @@ These methods are part of the [Hibernatable WebSockets API](/durable-objects/ref
...
+  - If `0`, or no value is given for the `timeout`, the previously set timeout (if any) will be unset.
+  - The maximum value of `timeout` is 604,800,000 ms -- i.e. 7 days.
```
Comment (Maddy-Cloudflare) [inline content/durable-objects/api/websockets.md]:
```suggestion
  - The maximum value of `timeout` is 604,800,000 ms, meaning 7 days.
```

**Review by Maddy-Cloudflare — CHANGES_REQUESTED** [review]

**Review by XoL1507 — COMMENTED** [review]
https://github.com/MystenLabs/sui/commit/fba929cc8a5dcb7ceb5663ceee43bba00b4670d1

**Review by Maddy-Cloudflare — APPROVED** [review]

**Review by jqmmes — APPROVED** [review]
LGTM

**Pull request merged**